### ISSUE TITLE
Error Italy download store No Library Record Found for user

### ISSUE THREAD

**Issue by Alessandro891** [openaudible/openaudible#733]
Message error: Download error: No Library Record Found for user.
the book is detected in the library but the download does not start.

**Comment by openaudible** [comment]
Is that for a purchased book or something included or free? If it is included with your membership, it can’t be backed up.  Only books you buy with cash or a credit can be backed up. Let me know!

**Comment by Alessandro891** [comment]
it is a book purchased with 1 monthly subscription credit, I also tried to import the.aax file, download from amazon.it and the same error always comes out.  Thank you

**Comment by openaudible** [comment]
It looks like all your books, including https://www.audible.it/pd/Fai-qualcosa-Audiolibri/8852155228 are free books, included with a monthly audible membership. So unfortunately, even if you bought it for a credit, it may now be considered free, and free books are not currently something we can back up. Sorry if there is confusion. I'm not totally sure if you can buy credits in Italy or if all books are part of the "all you can listen to" model. I show 3.5% of the non-us users are from Italy, but we haven't had a lot of feedback.

**Comment by Alessandro891** [comment]
you're right, my purchase makes all audiobooks free and in Italy it doesn't buy credits like in the United States, I noticed the service is different, here in Italy it only buys the subscription and not a single credit.  to get the aax file only with android i managed but it doesn't work.

is it possible to get a refund? as it is not possible to buy single audiobooks here but only free subscription

thank you very much and I apologize but I did not know it was not compatible in Italy that they do not sell credits.
